catalog/content/core/content-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -45,9 +47,17 @@
 					availability="optional">
 	</reference-list>
 
+	<reference-list id="preUpdateStoragePluginList" interface="ddf.content.plugin.PreUpdateStoragePlugin"
```

Comment:
It should be fine since `ContentFrameworkImpl` is already calling Pre/PostCreateStoragePlugins (none are enabled out of the box) and there haven't been any problems.